$OpenBSD$

XXX check after tdb gets an update (>1.2.7) if it is still needed
/usr/local/include/tdb.h:94: error: expected '=', ',', ';', 'asm' or '__attribute__' before 'struct'

--- metadata/rb-ext-db-key.h.orig	Wed Dec 21 16:01:41 2011
+++ metadata/rb-ext-db-key.h	Wed Dec 21 16:01:52 2011
@@ -30,6 +30,10 @@
 
 #include <glib-object.h>
 
+#ifndef _PUBLIC_
+#define _PUBLIC_
+#endif
+
 #include <sys/types.h>
 #include <tdb.h>
 
